### DISCUSSION TITLE
Enable Camera in Andorid Emulator / IOS

### DISCUSSION THREAD

**Discussion by renjuthoovara** [expo/expo#15477] [Archived: Questions]
Is there a way to enable Camera (Laptop Camera) in emulator using expo camera

**Comment by merodiro** [comment]
It seems that it was disabled intentionally in https://github.com/expo/expo/blob/master/packages/expo-camera/android/src/main/java/expo/modules/camera/CameraModule.kt#L89
I was able to enable it for android by patching it to remove this condition
